assets/js/wc-gateway-ppec-generate-cart.js

```diff
@@ -104,6 +104,10 @@
 				continue;
 			}
 
+			if ( -1 !== field_pairs[ i ].name.indexOf( 'addon-' ) ) {
+				field_pairs[ i ].name = field_pairs[ i ].name + field_pairs[ i ].value;
+			}
+
```

Comment:
> because a different plugin may require specific field names to be passed and this would break things.

Yes, that's what I fear. What if we check for duplicate keys only for a particular plugin like "product add-on"?

Reply:
> Yes, that's what I fear. What if we check for duplicate keys only for a particular plugin like "product add-on"?

That would work for now. But if `Product Add-on` makes any changes, we would need to reconfigure. We would also need to keep track of that plugin to see if anything has changed

Reply:
I added a new commit which should take care of this without any need for reconfiguration or tracking :)
The commit I added shows as "yesterday" because my vagrant/ubuntu time isn't up to date. I fixed that now.